app/services/weets.js

```diff
@@ -0,0 +1,18 @@
+const axios = require('axios');
+const logger = require('../logger');
+const { clientAPI, limit } = require('../../config/index.js').common;
+
+exports.getRandomQuote = async () => {
+    try {
+        const response = await axios.get(clientAPI, {
+            params: {
+                limit: limit
+            }
+        });
+        const { data } = response.data;
+        return data[0];
+    } catch (error) {
+        logger.error(error);
+        throw Error(error);
+    }
+}
```

Comment:
This is a minor thing, but remember to always leave an empty newline in all files